delivery_schenker/migrations/16.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Studio73 - Ferran Mora
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    for i in range(1, 23):
+        xml_id = "delivery_schenker.schenker_packaging_{}".format(str(i).zfill(2))
```

Comment:
Is this migration script working as expected? I would have thought this XML-ID doesn't exist anymore in post migration, unless such references are dropped right after the full module upgrade?